src/libraries/Extensions/Microsoft.Agents.Extensions.Teams.AI/Utilities/ResponseJsonParsers.cs

```diff
@@ -0,0 +1,169 @@
+﻿using System.Text.Encodings.Web;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace Microsoft.Agents.Extensions.Teams.AI.Utilities
+{
+    /// <summary>
+    /// LLM Response JSON Parsing Utilities
+    /// </summary>
+    public class ResponseJsonParsers
```

Comment:
Can be internal?  Compiles so... unless Cx uses it.

Reply:
Done